packages/appeals-service-api/src/lib/notify.js

```diff
@@ -103,10 +115,64 @@ async function sendEmail(appeal) {
   }
 }
 
+async function sendAppealSubmissionReceivedNotificationEmailToLpa(appeal) {
+  if (
+    !appeal ||
```

Comment:
Should be broken out into a "validate appeal" method. 

Also, is there a reason we are comparing type to undefined rather than using javascript falsy-ness?

Reply:
Sure, will extract 👍 

The `undefined` check was purposeful because of https://pins-ds.atlassian.net/browse/AS-1854?focusedCommentId=41833 - I actually strongly disagree with that, but as we _potentially_ accept empty strings, I can't check falsey (afaik). Or rather, I can, but then some are falsy, some are undefined... so for consistency I went with `undefined` only.

Reply:
Extracted as suggested:

https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/955/files#diff-ad5f0760e19d83abe499a1dec40798eca4065a35b873511af3e32d171970a39dR1

and

https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/955/files#diff-093b15b6daa1a4fad2febb382735ac58d695b2b6a80fd5055e1d392ef5967b83R1